src/object/srv_obj_migrate.c

```diff
@@ -2366,7 +2372,8 @@ migrate_obj_ult(void *data)
 	rc = migrate_one_epoch_object(oh, &epr, tls, arg);
 
 close:
-	dsc_obj_close(oh);
+	/* FIXME: should keep this handle open until all ULTs complete:
+	dsc_obj_close(oh);*/
```

Comment:
(style)  Block comments use * on subsequent lines
(style)  Block comments use a trailing */ on a separate line

Reply:
This leak is to check the issue here only. My assumption is we close this handle before all ULTs complete.